utils.py

```diff
@@ -61,3 +61,25 @@ def dict_to_df(data: str) -> DataFrame:
     df = pd.DataFrame.from_dict(json_file, orient='index', columns=indicators)
     df.index = df.index.map(int)
     return df
+
+def get_plot_indicators(config):
+    """
+    Method that initializes indicators
+    :param config: main configuration
+    :type config: dict
+    :return: None
+    :rtype: None
+    """
+
+    default_ind1 = ['ema5', 'ema21']
+    default_ind2 = ['volume']
+
+    try:
+        config["mainplot_indicator"]
+    except KeyError:
+        config["mainplot_indicator"] = default_ind1
```

Comment:
Ha, just to share [https://www.w3schools.com/python/ref_dictionary_setdefault.asp](https://www.w3schools.com/python/ref_dictionary_setdefault.asp)
```suggestion
        config.setdefault("mainplot_indicator", default_ind1)
```